src/utils/fetchJSON.ts

```diff
@@ -0,0 +1,20 @@
+export default async function fetchJSON<T>(
+  input: RequestInfo,
```

Comment:
Yeah, this one was interesting. I went with `input` because that's what the type declarations for `fetch`'s first parameter. I looked into why it's called `input`, and it appears to come from the [Fetch Standard document](https://fetch.spec.whatwg.org/#request-class):

```idl
typedef (Request or USVString) RequestInfo;

[Constructor(RequestInfo input, optional RequestInit init = {}),
 Exposed=(Window,Worker)]
interface Request {
```

Note that this is for the `Request` constructor, but it has the same parameters as `fetch`. In case you're curious, here's the data source for the TypeScript type declaration for `fetch`: https://github.com/microsoft/TSJS-lib-generator/blob/128463a8ecc3eb62f690d28560117ad248203720/inputfiles/idl/Fetch.widl#L102.